docs/_plugins/search_index.rb

```diff
@@ -64,15 +64,15 @@ def to_product_name(edition)
     selection_lambda = lambda {|_| false }
   end
 
-  editions = editions.select &selection_lambda
+  selected_editions = editions.select &selection_lambda
```

Comment:
Hmm.. I see only renamed variables. How does it solve the issue?

Reply:
The variable `editions` is also in the global scope. Since, we are assigning editions, I suspect that's what causing the issue. 

I ran the entire workflow with a fork of Spark NLP. I was not able to replicate the issue for the `zero_shot_ner_roberta`.

Reply:
I don't think it will help because we don't modify the variable value here.
Maybe it's an issue with incremental build?